command/command.go

```diff
@@ -147,3 +152,51 @@ func parseYAMLFile(pth string) (m *yaml.Node, retErr error) {
 	m, retErr = parseYAML(f)
 	return
 }
+
+func parseFile(pth string) (contents string, retErr error) {
+	f, err := os.Open(pth)
+	if err != nil {
+		retErr = fmt.Errorf("failed to open file: %w", err)
+		return
+	}
+	defer func() {
+		if err := f.Close(); err != nil && retErr == nil {
+			retErr = fmt.Errorf("failed to close file: %w", err)
+		}
+	}()
+	c, retErr := ioutil.ReadAll(f)
+	contents = string(c)
+	return
+}
+
+func removeNewLineChanges(beforeContent string, afterContent string) string {
+	lines := strings.Split(beforeContent, "\n")
+	edits := myers.ComputeEdits(span.URIFromPath("before.txt"), beforeContent, afterContent)
+	unified := gotextdiff.ToUnified("before.txt", "after.txt", beforeContent, edits)
+
+	editedLines := make(map[int]string)
+	for _, h := range unified.Hunks {
```

Comment:
It may be because I've been awake for 30 hours or because AST trees are the bane of my existence but... what exactly is this doing?

Reply:
Added a bunch of comments